src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx

```diff
@@ -157,8 +169,25 @@ export default function ContextualizedLocDetails() {
 
                                 <Col md={ 4 } className="closed-icon-container">
                                     <LocItemDetail
-                                        label="Requested by">{ locRequest.userIdentity?.firstName || "" } { locRequest.userIdentity?.lastName || "" }<br />
-                                        { locRequest.requesterAddress || "-" }
+                                        label="Requested by"
+                                    >
+                                        { locRequest.userIdentity?.firstName || "" } { locRequest.userIdentity?.lastName || "" }
+                                        {
+                                            locRequest.requesterAddress !== null && locRequest.requesterAddress !== undefined &&
```

Comment:
I think this shorter line does the same:
```suggestion
                                            locRequest.requesterAddress &&
```

Reply:
Unfortunately, this does not work here: JSX expects a boolean, not a "falsy" expression.